examples/dns/recordset.yaml

```diff
@@ -1,15 +1,15 @@
 apiVersion: dns.yandex-cloud.jet.crossplane.io/v1alpha1
 kind: Recordset
 metadata:
-  name: example-recordset
+  name: apps
 spec:
+  deletionPolicy: Orphan
   forProvider:
-    zoneIdRef:
-      name: example-zone
-    name: example-recordset
-    type: "A"
-    ttl: 200
     data:
-      - "10.1.0.1"
+      - '<ip-address>' # тут должен быть ip-адрес
```

Comment:
It's not valid filed value. It's beter to use valid values, and add descriptions in comments

Reply:
Fixed.